src/en/02_quick_start.md

```diff
@@ -1,27 +1,50 @@
 # Quick Start
 
-## Install guide
+## Install Quide
 
-Download PgManage distribution file for your platform from the CommandPrompt website: https://www.commandprompt.com/products/pgmanage/
+Download PgManage distribution file for your platform from the [Command Prompt website](https://www.commandprompt.com/products/pgmanage/).
 
 ### Linux
 
-PgManage for Linux is packaged in .AppImage format and does not require installation. Download the .AppImage format, make it executable and run.
+PgManage for Linux is packaged in `.AppImage` format and does not require installation. Download the `.AppImage` format, make it executable, and run it. For example, in an Ubuntu distribution:
+
+```
+chmod +x pgmanage-1.0.0a.AppImage
+/pgmanage-1.0.0a.AppImage
+```
 
 ### Windows
 
-Run pgmanage setup executable and follow the installation instructions.
+Run the PgManage setup executable and follow the installation instructions.
 
 ### Mac
 
-Download the DMG file and double click on it and drag PgManage icon to the Applications Folder icon.
+Download the `DMG` file and double click on it and drag the PgManage icon to the `Applications Folder` icon.
```

Comment:
I am not a native English speaker but this and .... and ... expression looks a bit weird to me.